mcs_benchmark_data/cli/cli.py

```diff
@@ -39,7 +40,10 @@ def __configure_logging(self, args):
 
     def main(self):
         args = self.__parse_args()
-        self.__commands[args.command](args)
+        for command in self.__commands.values():
+            command.add_arguments(arg_parser)

```

Comment:
You don't need this bit here, since you're already adding the arguments in parse_args (and they've already been parsed here).